BUILD

```diff
@@ -124,11 +124,24 @@ filegroup(
     visibility = ["//visibility:public"],
 )
 
+constraint_setting(name = "machine_size")
+
+# A machine with "high cpu count".
+constraint_value(
+    name = "highcpu_machine",
+    constraint_setting = ":machine_size",
+    visibility = ["//visibility:public"],
+)
+
+# The default RBE platform where most actions run on
 platform(
-    name = "rbe_ubuntu1604_with_network_and_privileged",
-    parents = ["@bazel_toolchains//configs/ubuntu16_04_clang/latest:platform"],
+    name = "rbe_ubuntu1604_default",
+    parents = ["@bazel_toolchains//configs/ubuntu16_04_clang/1.2:rbe_ubuntu1604"],
```

Comment:
you should still be able to point to latest?

Reply:
I don't find it all that useful to point to latest. Also, the corresponding docker container gcr.io/bazel-untrusted/ubuntu1604_bazel_rbe needs to be updated in sync anyway. So I think I'd prefer to specify the platform like this.

Reply:
sgtm